tests/unit/test_views_auth.py

```
@@ -497,112 +357,47 @@ def test_non_string_profile_fields_return_400(self, auth_client):
-    def test_social_account_uid_is_unique_per_provider(self, db):
-        """The same Google uid cannot be assigned to two different users."""
-        from allauth.socialaccount.models import SocialAccount
-
-        u1 = User.objects.create_user(username="g_user1", email="g1@example.com")
-        u2 = User.objects.create_user(username="g_user2", email="g2@example.com")
-        SocialAccount.objects.create(
-            user=u1, provider="google", uid="google-uid-003", extra_data={}
+        u = User.objects.create_user(
+            username="prof_user",
+            email="prof@example.com",
+            password="pass12345",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "password"

(S106)